packages/manager/apps/okms/src/data/hooks/useUpdateOkmsName.ts

```diff
@@ -56,24 +50,30 @@ export const useUpdateOkmsName = ({
       await queryClient.invalidateQueries({
         queryKey: okmsQueryKeys.listDatagrid,
       });
-      await queryClient.invalidateQueries({
-        queryKey: getServiceDetailsQueryKey(okmsId),
-      });
-      await queryClient.invalidateQueries({
-        queryKey: okmsQueryKeys.detail(okmsId),
-      });
+
+      // Wait for the renaming to be effective in the backend before refetching the data
```

Comment:
Ouch. I guess you did this because you where forced to, but : 
- what's happening here ? the user sees the name updated after 3s ?
- did we have issues with this before ? I don't remember we did.
- couldn't we fix this with an optimistic update on the detail, so the user sees the name changed, then we might trigger a delayed cache invalidation on the list

Also, no need to await cache invalidations.